library/core/src/intrinsics.rs

```diff
@@ -1388,6 +1388,21 @@ extern "rust-intrinsic" {
     /// [`f64::round`](../../std/primitive.f64.html#method.round)
     pub fn roundf64(x: f64) -> f64;
 
+    /// Returns the nearest integer to an `f32`. Rounds half-way to the nearest even
+    /// integer.
+    ///
+    /// The stabilized version of this intrinsic is
```

Comment:
It's not stable yet, so let's not call it `stabilized` yet.

Reply:
Ah that's a silly copy-paste error from me.